src/fragments/sdk/push-notifications/android/handle-fcm.mdx

```diff
@@ -8,6 +8,9 @@ By specifying this action, you can open the app when the user taps on the notifi
 
 **Adding a Receiver**
 
+
+>This receiver is no longer necessary and should be removed as of `com.amazonaws:aws-android-sdk-pinpoint:2.48.0`.
```

Comment:
```suggestion
<Callout>

This receiver is no longer necessary and should be removed as of `com.amazonaws:aws-android-sdk-pinpoint:2.48.0`

</Callout>
```

Reply:
I'd recommend going with a callout as @tjleing mentioned. Here's a comparison of the two.

### Current:
<img width="567" alt="Screen Shot 2022-06-10 at 4 49 46 PM" src="https://user-images.githubusercontent.com/52051793/173148210-6c62ae53-7a84-447f-ae27-372bd3f6193c.png">

### As Callout
<img width="589" alt="Screen Shot 2022-06-10 at 4 49 10 PM" src="https://user-images.githubusercontent.com/52051793/173148299-e42a4558-be43-4938-bd7b-44be42300d80.png">

Otherwise LGTM.

Reply:
Ah ok, worked with @tjleing and we had settled on `>` markdown callouts instead of `<Callout>` but browsing through the docs, it looks like `<Callout>` is more commonly used.